common/app/routes/challenges/components/classic/Classic.jsx

```diff
@@ -111,6 +141,13 @@ export class Challenge extends PureComponent {
         </Col>
         { this.renderPreview(showPreview) }
         <BugModal />
+        <ClassicModal
+          keyBindings={ keyBindings }
+          modalShow={ isChallengePassed }
+          modalTitle={ successMessage }
+          onModalClick={ submitChallenge }
+          onModalExit={ testsInvalid }
```

Comment:
A couple things about the property names

We know it is a modal, we don't need to prefix the names with `modal`

For booleans use the convention of naming as a question.

`modalShow` should be `isModalOpen`. and Since we know it is a modal it can be shortend to `isOpen`

onModelExit should be `closeModal`. Actions should be verbNouns where possible.

onModalClick should just be submitChallenge. Remember this is not a generic modal. It has a specific purpose.